src/constants.tsx

```diff
@@ -0,0 +1,17 @@
+/** Constant string literals in alphabetical order */
+// IDs and keys (user-invisible strings)
+export const FOLDER_ID_KEY: string = 'webmarkFolderId';
+export const LOAD_HERE_KEY: string = 'loadHere';
+export enum NotificationId {
+    PageAlreadyExists = 'Page Already Exists',
+    FolderEmpty = 'Folder Empty',
+    SaveSuccessful = 'Save Successful',
+};
+
+// to be used for i18n later (user-visible strings)
```

Comment:
are these json above and export statements below what you mean by "two sections?"

Reply:
Yeah, the two sections refer to this file and only the bottom section will be used to support i18n.